tests/individual-pool-search/test-books-pool.rb

```diff
@@ -40,7 +40,7 @@ def test_include(all,one)
     expect_status( 200 )
 
     # json_body[:facet_list][3][:buckets][0] returns FacetLibrary against book pool
-    numb_facetlibrary = json_body[:facet_list][3][:buckets][0][:count]
+    numb_facetlibrary = json_body[:facet_list][4][:buckets][0][:count]
```

Comment:
To avoid needing these changes in the future, you should expect the name of the facet to be the same, but the position int he response to be variable.